tensornetwork/block_sparse/block_tensor.py

```diff
@@ -1613,3 +1620,120 @@ def tensordot(tensor1: BlockSparseTensor,
       order=left_order + right_order,
       check_consistency=False)
   return res
+
+
+def svd(matrix: BlockSparseTensor,
```

Comment:
Unit tests please. :)

Reply:
sry, I had accidentally added the tests to the next PR block_tensor_test.py!

Reply:
there now!